src/api/integrations/companies-house/index.ts

```diff
@@ -0,0 +1,48 @@
+import axios from 'axios';
+import dotenv from 'dotenv';
+import { CompaniesHouseAPIResponse } from '../../types';
+
+dotenv.config();
+
+const username: any = process.env.COMPANIES_HOUSE_API_KEY;
+const companiesHouseURL: any = process.env.COMPANIES_HOUSE_API_URL;
+
+/**
+ * getCompanyHouseDetails
+ * makes companies house API call and returns company
+ * @param {String} companyNumber
+ * @returns {CompaniesHouseAPIResponse} CompanyHouseResponse object
+ */
+const companiesHouse = {
+  getCompanyHouseDetails: async (companyNumber: string): Promise<CompaniesHouseAPIResponse> => {
```

Comment:
`companyHouse` should always be "companies" to match the name of the API

I think this could just be called `get` though, what do you think? So consumption would be like this:

```js
const company = await companiesHouse.get('123456');
```

Reply:
done